wallet/lib/utils/constants.dart

```diff
@@ -151,30 +123,23 @@ const String kStripeSignoutJS =
 const String SOMETHING_WENT_WRONG = 'Something went wrong';
 
 //STRIPE ERROR STRING
-const String CREATE_PAYMENTINTENT_FAILED =
-    'Stripe PaymentIntent Creation Failed';
-const String GEN_PAYMENTRECEIPT_FAILED =
-    'Stripe Payment Receipt Generation Failed';
+const String CREATE_PAYMENTINTENT_FAILED = 'Stripe PaymentIntent Creation Failed';
+const String GEN_PAYMENTRECEIPT_FAILED = 'Stripe Payment Receipt Generation Failed';
 const String GEN_PAYOUTTOKEN_FAILED = 'Stripe Payout Token Generation Failed';
-const String GEN_REGISTRATIONTOKEN_FAILED =
-    'Stripe Registration Token Generation Failed';
+const String GEN_REGISTRATIONTOKEN_FAILED = 'Stripe Registration Token Generation Failed';
 const String GEN_UPDATETOKEN_FAILED = 'Stripe Update Token Generation Failed';
-const String GET_ACCOUNTLINK_FAILED =
-    'Stripe Get Connected Account Link Failed';
-const String GET_LOGINLINK_FAILED =
-    'Stripe Get Connected Account LOGIN Link Failed';
+const String GET_ACCOUNTLINK_FAILED = 'Stripe Get Connected Account Link Failed';
+const String GET_LOGINLINK_FAILED = 'Stripe Get Connected Account LOGIN Link Failed';
 const String PAYOUT_FAILED = 'Stripe Payout Request Failed';
-const String REGISTERACCOUNT_FAILED =
-    'Stripe Register Connected Account Failed';
+const String REGISTERACCOUNT_FAILED = 'Stripe Register Connected Account Failed';
 const String UPDATEACCOUNT_FAILED = 'Stripe Update Account Failed';
 const String IBC_HASH_UPDATE_FAILED = 'IBC hash info getting Failed';
```

Comment:
"IBC Hash Info failed" would be a better message

Reply:
I don't understand?

Reply:
Updated.